src/components/Signup/index.tsx

```diff
@@ -1,30 +1,95 @@
 import React from 'react';
+import { bool } from 'prop-types';
+import { booleanLiteral } from '@babel/types';
+import {Link, Redirect as redirect} from 'react-router-dom'
+import './styles/Signup.css'
```

Comment:
Since the style sheet is the same as login.css, we can maybe change the folder hierarchy such that login.css is available to both components. Maybe something like Auth/login, Auth/Signup, Auth/styles.css? and then that updates our login and signup imports to Auth/login or Auth/signup. (VS code has a plugin that fixes imports on its own when file path changes). Then import this stylesheet into both signup and login components. Does this make sense? Feel free to ask questions/suggest better ideas!